pkg/gateway/server/llmproxy.go

```diff
@@ -17,6 +18,15 @@ func (s *Server) llmProxy(req api.Context) error {
 		return types2.NewErrHTTP(http.StatusUnauthorized, fmt.Sprintf("invalid token: %v", err))
 	}
 
+	if token.UserID != "" {
+		promptTokensRemaining, completionTokensRemaining, err := s.client.RemainingTokenUsageForUser(req.Context(), token.UserID, 24*time.Hour, s.dailyUserTokenPromptTokenLimit, s.dailyUserTokenCompletionTokenLimit)
```

Comment:
nit: maybe the period should be be a constant (though I'm sure the compiler optimizes this).

Reply:
Done.